modules/op-understanding-credential-selection.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-understanding-credential-selection_{context}"]
+= Understanding credential selection
+
+A pipeline run or task run may require multiple authentications for accessing different Git repositories. You must annotate each secret to specify the domains for which {pipelines-shortname} can use the credential contained in a  secret.
```

Comment:
```suggestion
A pipeline run or task run might require multiple authentications to access different Git repositories. Annotate each secret with the domains where {pipelines-shortname} can use its credentials.
```

Reply:
For **May**, the IBM SG says `Use [may] only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.`

Reply:
About "for accessing" in the previous sentence, do you find that the "to <verb>" form of the phrase is simpler and easier to read than "for <verb>ing"? These are small choices, but they add up to a style that can be easier or more difficult for readers to consume.

Reply:
Should `domains` be singular?